build-tools/packages/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts

```diff
@@ -580,7 +580,7 @@ export const handlers: Handler[] = [
 					}
 				}
 
-				writeFile(file, JSON.stringify(json, undefined, 2) + newline);
+				updatePackageJsonFile(path.dirname(file));
```

Comment:
same issue here. Maybe making that call back optional wans't a great choice since this function just no-ops without it (plus sorting)